packages/broker/test/sequential/plugins/storage/CassandraNullPayloads.test.ts

```diff
@@ -34,50 +34,36 @@ const insertNullData = async (
         + '(stream_id, partition, bucket_id, ts, sequence_no, publisher_id, msg_chain_id, payload) '
         + 'VALUES (?, 0, ?, ?, 0, ?, ?, ?)'
     await cassandraClient.execute(insert, [
-        streamId, bucketId, new Date(), 'publisherId', 'msgChainId', null
+        streamId, bucketId, new Date(), '', '', null
     ], {
         prepare: true
     })
 }
 
-function buildMsg({
-    streamId,
-    streamPartition,
-    timestamp,
-    sequenceNumber,
-    publisherId = 'publisher',
-    msgChainId = '1',
-    content = {}
-}: {
-    streamId: string
-    streamPartition: number
-    timestamp: number
-    sequenceNumber: number
-    publisherId?: string
-    msgChainId?: string
-    content?: any
-}) {
-    return new StreamMessage({
-        messageId: new MessageIDStrict(streamId, streamPartition, timestamp, sequenceNumber, publisherId, msgChainId),
-        content: JSON.stringify(content)
-    })
-}
-
 async function storeMockMessages({
     streamId,
-    streamPartition,
     count,
     storage
 }: {
     streamId: string
-    streamPartition: number
     count: number
     storage: Storage
 }) {
     const storePromises = []
     for (let i = 0; i < count; i++) {
         const timestamp = Math.floor((i / (count - 1)) * (1E10))
-        const msg = buildMsg({ streamId, streamPartition, timestamp, sequenceNumber: 0, publisherId: 'publisher1' })
+
+        const msg = new StreamMessage({
+            messageId: new MessageIDStrict(
+                streamId,
+                0, //streamPartition
```

Comment:
Maybe could remove comments as the data is just mock data and the test doesn't rely on specific values.